process/README.md

```diff
@@ -108,7 +108,7 @@ Due to the increased community demands around KubeCon + CloudNativeCon (KCCN), t
 
 For each KubeCon + CloudNativeCon's (KCCN) standalone event (where KCCN is the focus, not a joint event with OSS Summit or other such conference) — currently Europe and North America — the following freeze is applied:
 
-__Within 6 weeks of the event__
+__Within 4 weeks of the event__
 
 * TOC members will not take on new sponsorship of applications for moving levels
 * Many activities occur before, during, and after KCCN. Postponing new sponsorship until after KCCN reduces the likelihood that kicking off the process is overcome by such activities.
```

Comment:
@linsun (@nikhita please help jog my memory) i think we talked about grouping this and the "within 2 weeks" up under a "within 4 week" heading. this makes it super simple and straightforward for projects and TOC to know what is happening when.

Reply:
Yes, we did have the grouping earlier - see within 2 weeks under https://github.com/cncf/toc/blob/a5c210b4dba9a3af281041b4f448d809ea7a4476/process/README.md#kubeconcloudnativecon-freeze